### DISCUSSION TITLE
proxy?

### DISCUSSION THREAD

**Discussion by GreenMystic** [TeamNewPipe/NewPipe#11527] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

since you guys have plan to bring newer newpipe 
is developer also have plan to add something like proxy?
(reference to libretube it's add proxy so libretube not communicating with yr directly)

🙏😅

**Comment by AudricV** [comment, marked as answer]
> - [x] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.

Duplicate of #992. Please search (better, with different keywords?) for existing discussions and issues the next time you want to open a new discussion. Thank you.

**Reply by GreenMystic** [reply]
my apologies i try better next time dev 🙏

(and hopefully I don't get banned here i sometimes do silly mistake due to search result is messed up😅)
